test/functional/feature_cltv.py

```diff
@@ -166,7 +167,7 @@ def run_test(self):
                     'allowed': False,
                     'reject-reason': expected_cltv_reject_reason,
                 }],
-                self.nodes[0].testmempoolaccept(rawtxs=[spendtx.serialize().hex()], maxfeerate=0),
+                self.nodes[0].testmempoolaccept(rawtxs=[ToHex(spendtx)], maxfeerate=0),
```

Comment:
Is there a reason to do this replacement? Now that python supports bin->hex by just appending `.hex()`, the helper isn't really needed anymore.

Personally I think both versions are fine and there isn't really a way to "protect" new code from using one or the other.

Reply:
Agree that the gain can be questioned -- I was also hesitating about this replacement, but then thought symmetry (i.e. why get rid of most deserialize() calls, but let the serialize() calls in) and the fact that it's shorter may be worth it.

Some tests already imported and used ToHex, but at some places still called serialize().hex(). At least mixing within the same test should be avoided IMHO, as it only creates confusion.

Maybe in the long-term, removing `ToHex` completely would be an alternative?

Reply:
Then my preference would be to remove `ToHex` to avoid future pull requests changing it back (or forth)